src/views/panel/panel.vue

```diff
@@ -70,9 +70,9 @@
 			User,
 			CreditCard
 		},
-		methods:{
-			handleTabClick(tab, event){
-				if(tab.index == 2){
+		methods: {
+			handleTabClick(tab, _event){
+				if(tab.index === Tabs.Me){
```

Comment:
This two cannot compare by '===', I guess they are different format. So I will change to '=='.